docs/cloud-dev-env.md

```diff
@@ -0,0 +1,31 @@
+# Cloud Development Environments
+
+Altis has experimental built-in support for [GitHub Codespaces](https://github.com/features/codespaces), a Cloud-based development environment, that makes it possible to spin up a complete development environment in seconds within your browser, with a full editor based on Visual Studio Code and a fully-featured environment for your project, without having to install anything on your local machine.
```

Comment:
We mention both a complete environment and a fully featured environment in this same paragraph so we can probably eliminate one and simplify it a bit.